app/components/candidate_interface/course_choices_review_component.html.erb

```diff
@@ -1,5 +1,51 @@
 <% @course_choices.each do |course_choice| %>
-  <div class='qa-application-choice-<%= course_choice.id %>'>
+  <% show_warning = course_choice.course_not_available? || course_choice.course_full? || course_choice.chosen_site_full? %>
+  <div class='qa-application-choice-<%= course_choice.id %> <%= show_warning ? 'review-warning'  : '' %>'>
+
+  <% if course_choice.course_not_available? %>
+    <div class='review-warning--primary-message'>
+      <p>You cannot apply to '<%= course_choice.course.name_and_code %>' because it is not running.</p>
+    </div>
```

Comment:
```suggestion
    <p class="app-review-warning--primary-message">You cannot apply to '<%= course_choice.course.name_and_code %>' because it is not running.</p>
```

This ^ combined with a change to the CSS, the div isn't doing anything for us here. :)